### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by Strflightmight09**

### CODE REVIEW

**Review by levyishai — CHANGES_REQUESTED** [review]
Replace all the Math.pow in the shooting calcs to x*x and make sure to rumble only when interrupted

File: src/main/java/frc/trigon/robot/misc/ShootingCalculations.java [hunk 1]
```diff
@@ -196,9 +196,9 @@ private Rotation2d calculateTargetPitchUsingProjectileMotion(double noteTangenti
         Logger.recordOutput("ShootingCalculations/NoteTangentialVelocity", noteTangentialVelocity);
         Logger.recordOutput("ShootingCalculations/ShooterNoteExitPointHeight", noteExitPointHeightDifferenceFromTarget);
         final double gForce = ShootingConstants.G_FORCE;
-        final double velocitySquared = Math.pow(noteTangentialVelocity, 2);
-        final double velocity4thPower = Math.pow(noteTangentialVelocity, 4);
-        final double distanceSquared = Math.pow(shooterNoteExitPointXYDistanceFromShootingTarget, 2);
+        final double velocitySquared = noteTangentialVelocity * noteTangentialVelocity;
+        final double velocity4thPower = noteTangentialVelocity * noteTangentialVelocity * noteTangentialVelocity * noteTangentialVelocity;
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/misc/ShootingCalculations.java]: multiply the velocity squared by itself

File: src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java [hunk 2]
```diff
@@ -110,14 +110,14 @@ public class ClimberConstants {
...
+            RIGHT_VISUALIZATION = new ClimberVisualization(
+            "RightClimberVisualization",
             BLUE,
             LIGHT_BLUE,
             RIGHT_CLIMBER_FIRST_JOINT_ORIGIN_POINT
     ),
-            LEFT_MECHANISM = new ClimberVisualization(
-                    "LeftClimberMechanism",
+            LEFT_VISUALIZATION = new ClimberVisualization(
+                    "LeftClimberVisualization",
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java]: just call these RightClimber and LeftClimber

File: src/main/java/frc/trigon/robot/subsystems/climber/ClimberVisualization.java [hunk 3]
```diff
@@ -135,9 +135,9 @@ private Rotation2d calculateStringAngle(Rotation2d climberAngle, double stringLe
 
     private Rotation2d calculateFirstJointPitch(double stringLength) {
         final double numeratorCalculation =
-                Math.pow(ClimberConstants.FIRST_JOINT_POSE_TO_STRING_CONNECTION_DISTANCE_METERS, 2)
-                        + Math.pow(ClimberConstants.FIRST_JOINT_POSE_TO_DRUM_DISTANCE_METERS, 2)
-                        - Math.pow(stringLength, 2);
+                ClimberConstants.FIRST_JOINT_POSE_TO_STRING_CONNECTION_DISTANCE_METERS * ClimberConstants.FIRST_JOINT_POSE_TO_STRING_CONNECTION_DISTANCE_METERS
+                        + ClimberConstants.FIRST_JOINT_POSE_TO_DRUM_DISTANCE_METERS * ClimberConstants.FIRST_JOINT_POSE_TO_DRUM_DISTANCE_METERS
+                        - stringLength * stringLength;
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/subsystems/climber/ClimberVisualization.java]: I would put correct brackets for more readability

File: src/main/java/frc/trigon/robot/subsystems/intake/IntakeCommands.java [hunk 4]
```diff
@@ -39,7 +39,8 @@ private static Command getCollectionCommand() {
                         },
                         (interrupted) -> {
                             RobotContainer.INTAKE.sendStaticBrakeRequest();
-                            RobotContainer.INTAKE.indicateCollection();
+                            if (interrupted)
+                                RobotContainer.INTAKE.indicateCollection();
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/subsystems/intake/IntakeCommands.java]: if it wasn't interrupted (it ended by isFinished) we'll want to indicate.

**Review by levyishai — CHANGES_REQUESTED** [review]

**Review by levyishai — APPROVED** [review]

**Review by Nummun14 — APPROVED** [review]

**Pull request merged**